tests/signup/signup.test.ts

```diff
@@ -54,6 +54,44 @@ test.describe('signup form tests', () => {
     await expect(page.getByText('Log out')).toBeVisible()
   })
 
+  test('sign up and login for a new account', async ({page}) => {
```

Comment:
adding a test with a lot of duplication. It creates a user with password `user_password` and then logs in with `user-password`. The test should fail but it passes because of the bug introduced previously. Hopefully the candidates realize about it maybe when they try to extract the password in a constant variable 🤷